examples/morphir-elm-projects/evaluator-tests/src/Morphir/Examples/App/ListTests.elm

```diff
@@ -77,4 +77,7 @@ listMapTest _ =
 listMapTest2 : () -> List Bool
 listMapTest2 _ =
     map not [True,False,True]
---expected = [False,True,False]
\ No newline at end of file
+--expected = [False,True,False]
+
+listAppend : List a -> List a -> List a
```

Comment:
Testing this requires testEvalMultiple (from the TypeChecker PR). It also may fail because the result type depends on input types; this should be supported, but if it doesn't work just disable the test and ping me.